app/views/games/index.html.erb

```diff
@@ -3,6 +3,9 @@
 <h2> 
 
   <% @games.each do |game| %>
+    <% if game.white != nil && game.black == nil %>
```

Comment:
This if logic wont be needed switch to use the `available` scope on the controller